src/Components/Analyzers/src/build/netstandard2.0/Microsoft.AspNetCore.Components.Analyzers.targets

```diff
@@ -0,0 +1,15 @@
+<Project>
+  <Target Name="_RemoveComponentAnalyzer" BeforeTargets="ResolveLockFileAnalyzers" Condition="'@(Analyzer->Count())' != '0'">
```

Comment:
I'll have to go look at the targets, but it is confusing that this is *before* resolving analyzers

Reply:
Hopefully the discussion elsewhere in the PR makes this clearer now.

Reply:
Yeah, I've also called this out in the comment that precedes this target. Should help.